.dockerignore

```diff
@@ -1,4 +1,7 @@
 .git
+!/.git/HEAD
+!/.git/master
```

Comment:
should be 
```suggestion
!/.git/refs/heads
```

Reply:
i think

Reply:
Yeah I was testing and I just noticed it, also added the test branch that is usually used in tests releases